CMakeLists.txt

```diff
@@ -96,39 +97,43 @@ set_option_if(ACTS_BUILD_PLUGIN_AUTODIFF ACTS_BUILD_EVERYTHING)
 include(GNUInstallDirs) # GNU-like installation paths, e.g. lib/, include/, ...
 include(ActsCompilerOptions) # default compile options
 include(ActsComponentsHelpers) # handle components via add_..._if commands
-include(TargetSourcesLocal) # needed only until we require CMake 3.13
 # place build products in `<build_dir>/bin` and `<build_dir>/lib` for simple use
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}")
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}")
 
+# dependency versions
+set(_acts_autodiff_version 0.5.11)
```

Comment:
If we bundle this isn't respected. This should be defined as before inside the corresponding if.

Reply:
I would prefer to keep the variables all in one place. While trying to update the version requirements, setting them all in different places was very confusing. Putting this all consistently in a single spot was my response. I extended the documentation to include this reasoning.

Reply:
Right, but in case you don't use `find_package`, this version not guaranteed to be the one that's used. Do think that's not a problem we care about?

Reply:
I updated the documentation to clarify that these are the minimal version requirements, not the actual version used. E.g. for the `ActsConfig.cmake`, the found version is used, not the one specified in these variables. This is also true for the dependencies added via `add_subdirectory`.

Reply:
Ok.